### ISSUE TITLE
Version 1 of data submission does not download 

### ISSUE THREAD

**Issue by jleighherzog** [CDL-Dryad/dryad-product-roadmap#1220]
**Bug description:**
Unable to download the first published version of this data submission: 

Title: "Data from: The future of food from the sea"
DOI: 10.25349/D96G6H

_Note: The author, Tracy Teal, and I have all have experienced the same result when attempting to download the first published version._ 

**Steps to reproduce:**
There are two published versions of this data submission. When the "Download Dataset" menu is expanded to view the first version of the dataset (published on July 29, 2020; 1.32 GB), the current version is downloaded. The previously published version isn't available.  

**Expected behavior:**
The first published version should download when the file is directly selected. 

HEADS-UP: There are copyright issues with version 1 of this submission. This means, if/when you fix the issue, I will follow up to submit another ticket to modify the files included in version 1.

**Comment by sfisher** [comment]
Hi @jleighherzog ,

I'm not really sure what the author would like to do with this one since it sounds like it is complicated.  I've only investigated a little and haven't done anything yet and Daniella hasn't moved it over for us to work on yet.

As far as it downloading the correct items, it looks to me like it is operating correctly for the files it shows in the UI.  This dataset has 6 versions.  (See the activity log at https://datadryad.org/stash/ds_admin/activity_log/45165?utf8=%E2%9C%93 ).  From the logs, only versions 5 and 6 were published.  Everything before that wasn't published and so doesn't show up to be downloaded since only published datasets are show to the public.

This is the file in version 6.
![Screen Shot 2021-04-20 at 12 48 52 PM](https://user-images.githubusercontent.com/105433/115456688-4d7d5780-a1d8-11eb-90c7-373e1e234021.png)

This is the file in version 5.
![Screen Shot 2021-04-20 at 12 49 17 PM](https://user-images.githubusercontent.com/105433/115456770-6554db80-a1d8-11eb-9093-cc16bf7d155b.png)

This is the file in version 4.
![Screen Shot 2021-04-20 at 12 49 44 PM](https://user-images.githubusercontent.com/105433/115456801-6ede4380-a1d8-11eb-88b7-a7a18b66c8ff.png)

When I download the two versions that show up (July 29, 2020 and April 18, 2021) they definitely have different contents since they are very different sizes (1.3 GB for version 5 and 377 MB for version 6).

Version 4, which was never published and was updated on July 27, 2020) is a very similar size to version 5, but it has different contents as seen in the screenshot above since the numbers following the size are a hash digest of the contexts.  The hash is like a fingerprint of the file and if the hash is different then the contents are certainly different.

So, IDK if the user was hoping to use Version 4 as their first version or they don't want to see old versions at all or what will be the best for this situation.

We cannot really go back and modify any old versions while retaining current versions since all new versions depend on old ones.  We could unhide version 4 and make it into the 1st published version, even though it wasn't originally published, if we wanted to, though.

Or if they have data that doesn't belong to be public we can hide it or other things.

**Comment by jleighherzog** [comment]
Ok. I still have strange things happening on my end. If I select the files individually and download them, they are the same. When I select 'Download Dataset', the previous version is available, not the current version. Which isn't how that feature is supposed to work. 

A previous version contains copyrighted data that needs to be removed/hidden. It's tough for me to decipher which file that is though because the download options aren't acting right. 

So, I've attached a screen capture of the version that, ideally, should be hidden. I expect it's the version that precedes the current one. Version 5? Can you confirm and, if possible, hide it? If it's not possible to hide, any other options? 

Thanks for your effort on this! 

<img width="657" alt="Screen Shot 2021-04-22 at 4 54 52 PM" src="https://user-images.githubusercontent.com/76497320/115790550-7fcfb600-a38c-11eb-8bb7-01835e8a3546.png">

**Comment by sfisher** [comment]
Thanks, Jess.  This makes more sense based on your screenshot.  Sorry I didn't understand right away.  It looks like the old files are coming in from the "Download Dataset" link rather than something inside the zip in the download under each version date.

This is a problem caused when the Merritt team did a migration and some older files are still showed up that shouldn't because of something that happened in the migration.  They have a plan to correct it very soon for the few datasets that were affected.

In the meantime, I've manually submitted another version in order to manually remove the extra files that showed up.  It sounds like you didn't like the download under July 29, 2020, so I disappeared it.  Take a look to be sure this is the desired state.  If you want July 29, 2020 back, I can bring it back from the dead.   🧛   Just let me know.
